From 15e3db04f60ee1a35bd616b8a78ae3e4f85ef286 Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <icenowy@aosc.io>
Date: Fri, 5 May 2017 16:17:03 +0800
Subject: [PATCH 93/94] drm/sun4i: prepare for vblanks for two pipelines' CRTCs

When we initializing the drm vblank, we did it only for one CRTC, which
will lead to the second CRTC fail to do vblank when two pipelines are
all enabled.

Change the value to 2, in order to satisfy the situation that has 2
CRTCs.

Signed-off-by: Icenowy Zheng <icenowy@aosc.io>
---
 drivers/gpu/drm/sun4i/sun4i_drv.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/sun4i/sun4i_drv.c b/drivers/gpu/drm/sun4i/sun4i_drv.c
index f5f5ccb5344f..53b94e82fba9 100644
--- a/drivers/gpu/drm/sun4i/sun4i_drv.c
+++ b/drivers/gpu/drm/sun4i/sun4i_drv.c
@@ -111,7 +111,7 @@ static int sun4i_drv_bind(struct device *dev)
 	}
 
 	/* drm_vblank_init calls kcalloc, which can fail */
-	ret = drm_vblank_init(drm, 1);
+	ret = drm_vblank_init(drm, 2);
 	if (ret)
 		goto free_mem_region;
 
-- 
2.12.2

